tests/unit/test_cli/test_core_commands.py

```diff
@@ -48,6 +48,9 @@ def mock_show_app_info(mocker: MockerFixture) -> MagicMock:
         (False, None, 2),
     ],
 )
+@pytest.mark.parametrize(
+    "ssl_certfile, ssl_keyfile, create_devcert", [(None, None, False), ("cert.pem", "key.pem", True)]
+)
```

Comment:
This should go into a separate test case, as adding this parametrization adds another 128 test cases. This test case should only have options that are known to / could interfere with each other, since we want to test all combinations of those. Since the certs don't interact with anything else, they don't need to be included in here.